### ISSUE TITLE
Flathub verification

### ISSUE THREAD

**Issue by alcir** [AlizaMedicalImaging/AlizaMS#22]
@issakomi, since AlizaMS is available on Flathub and you are involved in the Flatpak package, it would be nice to have the checkmark under an application name on Flathub. The process looks pretty easy: https://flathub.org/about

**Comment by issakomi** [comment]
Thank you very much! I didn't know. Good idea.

![fh1](https://user-images.githubusercontent.com/1494580/233857574-9c5e7f4c-0963-4e41-a641-b261a94544d5.jpg)

**Comment by issakomi** [comment]
@alcir 

I have updated Flathub package, it is now with ITK 5.3.

BTW, I have seen the RPM package, thank you!
One minor comment about
`-DALIZA_CXX_STANDARD:STRING=14`
this flag may be not always good, latest compiler use C++17 by default already. Generally this flag is good only if there are issues, e.g. old GCC-4.8 will not build without standard forced to C++11 (minimum for Aliza and ITK 5.0). Current ITK 5.3 release works only with C++14, next ITK release will be C++17, master branch is already C++17. Qt6 requires C++17 too. IMHO, the best strategy would be to try without the flag and use as workaround if there are issues or warnings, it depends on compiler.

BTW, next release 1.9.1 will not take so long as this one. I have already fixed one very minor issue and I am waiting for
next DICOM release to support new IOD (Photoacoustic Image).

Another question is OpenGL context, Core Profile or Compatibility. I am not 100% sure, Compatibility may work with older hardware (OpenGL 3.0 from 2008), Core requires OpenGL 3.2 from 2009. The issue maybe that some drivers may drop Compat. Profile. E.g. on macOS it not possible any more to use Compat., only Core Profile 3.2 / GLSL 1.5. I am not sure, I have tested with old Intel chip, both works everywhere. The option for Qt5 is
`-DALIZA_QT5_COREGL:BOOL=ON`
For Qt6 Core Profile is taken by default. Just FYI.

Thank you and sorry for long message.
